File: twiml/voice/conference/conference-6/conference-6.5.x.cs
using Twilio.TwiML;
using Twilio.TwiML.Voice;
using System;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial();
        dial.Conference("NoMusicNoBeepRoom", beep: "false",
            waitUrl: new Uri("http://your-webhook-host.com"),
            startConferenceOnEnter: true, endConferenceOnExit: true);
        response.Append(dial);

        Console.WriteLine(response.ToString());;
    }
}

File: twiml/voice/conference/conference-6/conference-6.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
const dial = response.dial();
dial.conference({
    beep: false,
    waitUrl: 'http://your-webhook-host.com',
    startConferenceOnEnter: true,
    endConferenceOnExit: true
}, 'NoMusicNoBeepRoom');

console.log(response.toString());
